unicode_gen.c

```diff
@@ -1019,6 +1023,27 @@ void find_run_type(TableEntry *te, CCInfo *tab, int code)
             return;
         }
 
+        if (ci->u_len == 2 && ci->u_data[1] == 0x399 &&
+            ci->l_len == 1) {
+            len = 1;
+            while (code + len <= CHARCODE_MAX) {
+                ci1 = &tab[code + len];
+                if (!(ci1->u_len == 2 &&
+                      ci1->u_data[1] == 0x399 &&
+                      ci1->u_data[0] == ci->u_data[0] + len &&
+                      ci1->l_len == 1 &&
+                      ci1->l_data[0] == ci->l_data[0] + len))
+                    break;
+                len++;
+            }
+            te->len = len;
+            te->type = RUN_TYPE_U2_399_EXT2;
+            te->ext_data[0] = ci->u_data[0];
+            te->ext_data[1] = ci->l_data[0];
+            te->ext_len = 2;
+            return;
+        }
+
```

Comment:
Copy and pasted from the one below except there is no `F` component

Reply:
That means the if statement below is dead code now, right? Less specific subsumes more specific.

It's probably okay if you swap them. The code itself looks correct at a superficial glance.